code/DDSCodeTester.cpp

```diff
@@ -3969,372 +3969,518 @@ class CustomizedDataReaderListener : public DataReaderListener
 
 void dds_qos_examples()
 {
+    // Taken out of the examples to avoid bloating them
+    DomainParticipantFactory* factory_ = DomainParticipantFactory::get_instance();
+    DomainParticipant* participant_ =
+            DomainParticipantFactory::get_instance()->create_participant(0, PARTICIPANT_QOS_DEFAULT);
+    Subscriber* subscriber_ =
```

Comment:
Should variables local to the function end with a `_`? We usually use that nomenclature to indicate they are protected method attributes.

Reply:
After reading further, I've realized you use the notation to keep track of which variables are shared throughout the function and which ones have a reduced scope. I like it.

Reply:
Yes, it also replicates the structure followed in Fast DDS examples, where every entity is a private member of a Class. It's just to keep some coherence with the examples.